<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoRecordItemWidget</class>
 <widget class="QWidget" name="VideoRecordItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="cover_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>235</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>3</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173, 127, 168);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	font: 15pt &quot;Ubuntu&quot;;
	color: rgb(255, 255, 255);
	background:transparent;
</string>
   </property>
   <property name="text">
    <string>2023-10-13 11:59:30</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	font: 15pt &quot;Ubuntu&quot;;
	color: rgb(255, 255, 255);
	background:transparent;
</string>
   </property>
   <property name="text">
    <string>已观看:</string>
   </property>
  </widget>
  <widget class="QLabel" name="percentNum_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	font: 15pt &quot;Ubuntu&quot;;
	color: rgb(255, 255, 255);
	background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
